src/pages/ApproveAction/components/DetailedCardWrapper.tsx

```diff
@@ -0,0 +1,43 @@
+import { PropsWithChildren } from 'react';
+import { Card, Stack } from '@avalabs/core-k2-components';
+
+export const DetailedCardWrapper = ({
+  onClick,
+  children,
+  isFirst,
+  isLast,
+}: PropsWithChildren<{
+  onClick?: () => void;
+  isLast: boolean;
+  isFirst: boolean;
+}>) => {
+  return (
+    <Card
+      sx={{
+        display: 'flex',
+        width: 320,
+        flex: '0 0 auto',
+        backgroundColor: 'grey.850',
+        ml: isFirst ? 3.5 : 0,
+        mr: isLast ? 3.5 : 0,
+        position: 'relative',
```

Comment:
I know designs are like this so no action just a note. We are getting sooo far off from the component library.